app.js

```diff
@@ -3,12 +3,14 @@ const app = express()
 const {
   getTopics,
   getArticles,
-  getArticleByID
+  getArticleByID,
+  getCommentsByArticleId
 } = require("./controllers")
 
 app.get("/api/topics", getTopics)
 app.get("/api/articles", getArticles)
 app.get("/api/articles/:article_id", getArticleByID)
+app.get("/api/articles/:article_id/comments", getCommentsByArticleId)
 
 app.use("/*", (req, res, next) => {
   res.status(404).send({ msg: "Path not found" })
```

Comment:
I think you should start to put your errors into an errors.controller file in order to keep app.js tidy.